documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -133,3 +133,22 @@ The date MAY optionally be followed by one of:
 * 2016-07-04-rc0 - Just use "rc", not "rc" + number
 
 Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [RPC](#rpc-violations): [Errors](#rpc-errors) or [Warnings](#rpc-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
+
+### <a name="M3016" />M3016 PropertiesNamesCamelCase
```

Comment:
we've documented this rule at https://github.com/Azure/autorest/blob/master/docs/developer/validation-rules/camel-casing-validation-rules.md should we keep the location in autorest (that's where the rules are implemented) and have one page per rule as in the "developer" folder in autorest/docs?